wandb/sdk/launch/launch.py

```diff
@@ -161,6 +163,10 @@ def run(
     if config is None:
         config = {}
 
+    if "docker" in config:
+        docker_args.update(config["docker"])  # userprovided args override
```

Comment:
Are docker_args unused now? It seems like this  and the following line could be removed?

Reply:
in theory they are passed thru to the `docker run` command  via `get_full_command`

Reply:
Got it